File: Java/Mahasiswa.java
public class Mahasiswa extends Akademik{
    private String nim;
    private String prodi;
    private String fakultas;

    //Constructors
    public Mahasiswa(){
        this.nim = "";
        this.prodi = "";
        this.fakultas = "";
    }

    public Mahasiswa(String nim, String prodi, String fakultas){
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    //Getter NIM
    public String getNIM(){
        return this.nim;
    }
    //Setter NIM
    public void setNIM(String nim){
        this.nim = nim;
    }


    //Getter Prodi
    public String getProdi(){
        return this.prodi;
    }
    //Setter Prodi
    public void setProdi(String prodi){
        this.prodi = prodi;
    }

    //Getter Fakultas
    public String getFakultas(){
        return this.fakultas;
    }
    //Setter Fakultas
    public void setFakultas(String fakultas){
        this.fakultas = fakultas;
    }

    public void setAll(String prodi, String fakultas){
        this.prodi = prodi;
        this.fakultas = fakultas;
    }
    //Method cari NIM
    public int cariNim(String nim){
        
        if(this.nim.equals(nim)){
            return 1;
        }else{
            return 0;
        }
    }

}

File: PHP/Mahasiswa.php
<?php
require('Akademik.php');
class Mahasiswa extends Akademik
{
	//atribut
	private $nim = "";
	private $fakultas = "";
	private $prodi = "";

	//constructor
	public function __construct($nim = "",  $prodi = "", $fakultas = "")
	{
		$this->nim = $nim;
		$this->fakultas = $fakultas;
		$this->prodi = $prodi;
	}

	//NIM
	public function setNim($nim)
	{
		$this->nim = $nim;
	}

	public function getNim()
	{
		return $this->nim;
	}

    //fakultas
	public function setFakultas($fakultas)
	{
		$this->fakultas = $fakultas;
	}

	public function getFakultas()
	{
		return $this->fakultas;
	}

    //prodi
	public function setProdi($prodi)
	{
		$this->prodi = $prodi;
	}

	public function getProdi()
	{
		return $this->prodi;
	}

	//destructor
	public function __destruct()
	{
	}
}

?>